### PR TITLE
Update the hardware requirements

### PR DESCRIPTION
**Pull request by TheFrenchGhosty**
Note: I'm not sure 1GB of ram is enough to run Invidious

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
 
 Note: Any [PaaS](https://en.wikipedia.org/wiki/Platform_as_a_service) or [SaaS](https://en.wikipedia.org/wiki/Software_as_a_service) provider/software (Heroku, YunoHost, Repli...) are unsupported. Use them at your own risk. They **WILL** cause problems with Invidious and might even suspend your account for "abuse" since Invidious is heavy, bandwidth intensive and technically a proxy (and most providers don't like them). If you use one and want to report an issue, please mention which one you use.
 
 
+## Hardware requirements
+
+Running Invidious requires at least 1GB of free RAM (We recommend to have at least 2GB installed). We recommend public instances to have at least 4GB of RAM, 2vCPU, a 200 mbps link and 20TB of traffic (unlimited traffic is preferred).
+
+Compiling Invidious requires at least 2GB of free RAM (We recommend to have at least 3GB installed).
+If you have less (e.g on a cheap VPS) you can setup a SWAP file or partition, so the combined amount is >= 3GB.
```
Comment (SamantazFox) [inline docs/installation.md:13]: I think it's 2.5G now, so we should recommend 4G of RAM.

File: docs/installation.md [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
-If you have less (e.g on a cheap VPS) you can setup a SWAP file or partition, so the combined amount is >= 3GB.
-
 After installation take a look at the [Post-install steps](#post-install-configuration).
 
 Note: Any [PaaS](https://en.wikipedia.org/wiki/Platform_as_a_service) or [SaaS](https://en.wikipedia.org/wiki/Software_as_a_service) provider/software (Heroku, YunoHost, Repli...) are unsupported. Use them at your own risk. They **WILL** cause problems with Invidious and might even suspend your account for "abuse" since Invidious is heavy, bandwidth intensive and technically a proxy (and most providers don't like them). If you use one and want to report an issue, please mention which one you use.
 
 
+## Hardware requirements
+
+Running Invidious requires at least 1GB of free RAM (We recommend to have at least 2GB installed). We recommend public instances to have at least 4GB of RAM, 2vCPU, a 200 mbps link and 20TB of traffic (unlimited traffic is preferred).
```
[the remark this answers is not in the document]
Comment (TheFrenchGhosty) [reply docs/installation.md]: Done

File: docs/installation.md (hunk 1 above)
Comment (TheFrenchGhosty) [reply docs/installation.md]: Done

File: docs/installation.md [hunk 3]
```diff
@@ -1,13 +1,18 @@
...
-If you have less (e.g on a cheap VPS) you can setup a SWAP file or partition, so the combined amount is >= 3GB.
-
 After installation take a look at the [Post-install steps](#post-install-configuration).
 
 Note: Any [PaaS](https://en.wikipedia.org/wiki/Platform_as_a_service) or [SaaS](https://en.wikipedia.org/wiki/Software_as_a_service) provider/software (Heroku, YunoHost, Repli...) are unsupported. Use them at your own risk. They **WILL** cause problems with Invidious and might even suspend your account for "abuse" since Invidious is heavy, bandwidth intensive and technically a proxy (and most providers don't like them). If you use one and want to report an issue, please mention which one you use.
 
 
+## Hardware requirements
+
+Running Invidious requires at least 512MB of free RAM, as long as the post-install configuration is done (We recommend to have at least 2GB installed). Public instances should ideally have at least 4GB of RAM, 2vCPU, a 200 mbps link and 20TB of traffic (unlimited traffic is preferred).
```
Comment (SamantazFox) [inline docs/installation.md:10]: "Running Invidious requires at least 512MB of free RAM (So ~2G installed on the system), as long as the instance is restarted regularly, as recommended in [the post-install configuration]."

Sounds better, imo	




"

Comment (TheFrenchGhosty) [reply docs/installation.md]: Done.

**Pull request merged**